data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/BeaconRestApi.java

```diff
@@ -13,14 +13,22 @@
 
 package tech.pegasys.artemis.beaconrestapi;
 
+import static org.eclipse.jetty.http.HttpStatus.NOT_FOUND_404;
```

Comment:
do we need to use jetty 404?

Reply:
hrm no that's what it imported, i'll see if there's a better option